code/obj/nuclearreactor/turbine.dm

```diff
@@ -0,0 +1,219 @@
+// ----------------------------------------------------- //
+// Defintion for the turbine used by the nuclear reactor
+// This is where the power comes from
+// ----------------------------------------------------- //
+
+
+/obj/machinery/atmospherics/binary/reactor_turbine
+	name = "Gas Turbine"
+	desc = "A large turbine used for generating power using hot gas."
+	icon = 'icons/obj/large/96x160.dmi'
+	icon_state = "turbine_main" //TODO make rotated states of this
```

Comment:
TODO as in this PR?

Reply:
Nah, this was an early thing that has since been abandoned. will remove.